css/style.css

```diff
@@ -321,7 +330,7 @@ button:hover {
 	opacity: 1;
 }
 
-.card .card-controls .space {
+.space {
```

Comment:
`<space>` element is not needed. Instead, you should add `margin-left: auto;` to `#board-header-controls`